articulate/orson
# Added tab traps to orson to control element focus when in full screen.

## Description
This addresses the issue found where users could not cycle tab through the video controls when viewing a video in full screen.

P.S. I do not know the other contexts in which to solve for, but I did check in Rise 360 and this seems to work. Shoutout to @evilsoft !!! Who always knows the solution! I didn't even know tab trapping was a thing! I hope I implemented it well.

## Links
Link to issue: https://github.com/articulate/rise-frontend/issues/5290
Link to frontend PR: https://github.com/articulate/rise-frontend/pull/5289

## Testing
Here's how to test it:
1. In Rise 360, go to a course, and add a video block by clicking Multimedia > Video
2. Open the video in fullscreen and tab through the video controls
3. Verify that the tabbing cycles all the way through, and that the focus never goes outside the video modal when in full screen
4. Also validate that the tab trapping goes away when the video is not in full screen mode, and that the focus continues on as normal as seen in production


[Code review guide](https://www.notion.so/Code-Review-063e76ea312742008a8dfb758c661d02)
